extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/internal/SonosAudioSink.java

```diff
@@ -92,9 +93,9 @@ public void process(AudioStream audioStream) throws UnsupportedAudioFormatExcept
                         logger.warn("Sonos speaker '{}' is not initialized - status is {}", handler.getThing().getUID(),
                                 handler.getThing().getStatus());
                     } else if (AudioFormat.WAV.isCompatible(format)) {
-                        handler.playNotificationSoundURI(new StringType(url + ".wav"));
+                        handler.playNotificationSoundURI(new StringType(url + FileAudioStream.WAV_EXTENSION));
                     } else if (AudioFormat.MP3.isCompatible(format)) {
-                        handler.playNotificationSoundURI(new StringType(url + ".mp3"));
+                        handler.playNotificationSoundURI(new StringType(url + FileAudioStream.MP3_EXTENSION));
```

Comment:
Hm ... probably not.
The constant(s) is/are used in the [getAudioFormat(File file)](https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/FileAudioStream.java#L47) method to determine the audio format. Maybe we should refactor those lines to compare file extensions using the [FilenameUtils.getExtension](http://commons.apache.org/proper/commons-io/javadocs/api-2.5/org/apache/commons/io/FilenameUtils.html#getExtension(java.lang.String)) method from Apache Commons IO. And additionally use `FilenameUtils.EXTENSION_SEPARATOR_STR` to concat again. wdyt?